mcp/src/utils/http.ts

```diff
@@ -19,47 +19,52 @@ async function performRequest({
   method,
   body,
 }: {
-  options: CourierClientOptions;
+  options?: CourierClientOptions;
   route?: string;
   url?: string;
   method: HttpMethod;
   body?: any;
 }): Promise<Response> {
 
   // Validate the options
-  if (!options.apiKey) {
+  if (options && !options?.apiKey) {
```

Comment:
Are there cases where it makes sense to initialize the server without options (and without an API key)? Could options stay required?